### ISSUE TITLE
obsolete GO:0016591 (RNAP II holoenzyme) term

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#10556]
Hi,

Harold and I were talking today and ended looking at this term. There are both some minor issues and some major issues (which I've put in SF before). At a minimum, we should fix the minor issues. I'll summarize the major issues also since I think we should seriously consider obsoleting this term.

Minor issues: There are a couple RNAP III txn factor complex terms ("TFIIIC-TOP1-SUB1 complex" and "transcription factor TFIIIE complex") that have been given "part_of" relationships to the RNAP II holoenzyme term. This seems unlikely, so probably these should be removed.

Major issues (reasons to consider obsoleting this term)
1. When I looked at this previously, I was unable to determine an unambiguous composition for a RNAP II holoenzyme. The phrase "RNAP II holoenzyme" is used by lots of people for many different "purifications" making it impossible to define a consistent composition.
1. The part_of relationships are probably not correct as things like TFIIA, TFIID, etc. can exist independently, so it is not correct to say that when TFIID exists it is part of "RNAP II holoenzyme".
2. The has_part relationship will not be correct either because there does not seem to be a consistent composition for "RNAP II holoenzyme" (see #1).

thanks,

-Karen

Reported by: krchristie

Original Ticket: [geneontology/ontology-requests/10360](https://sourceforge.net/p/geneontology/ontology-requests/10360)

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete GO:0016591 DNA-directed RNA polymerase II, holoenzyme'. The reason for obsoletion is that the composition of this complex cannot be clearly defined. 

Annotations should be reviewed. 

EXP annotations


No InterPro mappings 


Comments can be added to the ticket: https://github.com/geneontology/go-ontology/issues/10556

We are opening a comment period for this proposed obsoletion. We'd like to proceed and obsolete this term on July 9, 2018. Unless objections are received by July 9, 2018, we will assume that you agree to this change.

Thanks,

Pascale

**Comment by pgaudet** [comment]
OK with you @krchristie ?

**Comment by pgaudet** [comment]
part_of children can go to the parent: 'GO:0055029 nuclear DNA-directed RNA polymerase complex'

* GO:0016593 Cdc73/Paf1 complex
*  GO:0070847 core mediator complex
*  GO:0005665 DNA-directed RNA polymerase II, core complex
* GO:0005675 holo TFIIH complex
*  GO:0032039 integrator complex
*  GO:0034740 TFIIIC-TOP1-SUB1 complex
*  GO:0005672 transcription factor TFIIA complex
*  GO:0005669 transcription factor TFIID complex
* GO:0005673 transcription factor TFIIE complex
*  GO:0005674 transcription factor TFIIF complex
*  GO:0070264 transcription factor TFIIIE complex
*  GO:0033276 transcription factor TFTC complex 

OK ?

**Comment by krchristie** [comment]
I do not think that any of the part of children of holoenzyme can go to the parent term 'GO:0055029 nuclear DNA-directed RNA polymerase complex' because they are NOT part_of the polymerase enzyme and because we do NOT want to be inferring that these complexes have the molecular function of polymerase activity when they do not have or contribute to this activity. 

These other complexes are involved in various other functions in the process of transcription from RNAP II promoter, but they definitely do NOT have RNA polymerase activity.

**Comment by pgaudet** [comment]
So, what we have now is incorrect: 
<img width="473" alt="image" src="https://user-images.githubusercontent.com/4782928/42176650-f1952e5e-7e29-11e8-88e1-e84143ffd945.png">



where should they go then ? Just nuclear complexes ?

**Comment by krchristie** [comment]
What we have been saying in GO for part_of relations to the RNAP II holoenzyme term has been untrue for many years. It is not true to say that these complexes are ALWAYS part of holoeyzme, nor would it be true to say that holoeyzme ALWAYS has_part these complexes. 

Some of these already have other is_a parentage and may not require any additional is_a parentage. I'll leave it to you to decide if you want to add additional parentage specifying a subcellular localization.

**Comment by pgaudet** [comment]
@krchristie in your original comment (which I may have poorly copied), you write "When I looked at this previously, I was unable to determine an unambiguous composition for a RNAP II holoenzyme. The phrase "RNAP II holoenzyme" is used by lots of people for many different "purifications" making it impossible to define a consistent composition." 

Are you OK with obsoletion? 

I'll look for what to do with the children later - it won't be worse than what we have now). 

Thanks, Pascale

**Comment by srengel** [comment]
just saw the obsoletion email come through, but it's not clear to me from that email (or from this ticket) what we should do with these annotations. ...sounds like maybe it's still under discussion?

**Comment by krchristie** [comment]
At one point in time, during one of the many previous times this has been discussed, I think that there was some idea that the term is useful, despite the fact that the composition cannot be defined unambiguously. While there are likely many different holoenzymes, even in the same cell to target different promoter types, holoenzyme is often used to describe the complex that is active to begin transcription from a specific promoter. If we feel that it is useful to capture this, then the term could remain, but we would need to remove all those part_of relationships as I think it is also clear that there are enough variations in the composition of holoenzyme for either the current part_of relationships currently between the holoenzyme term and the various subcomplexes such as TFIIA, , or the alternately proposed has_part relationships, to always be true in either direction.

@ukemi - Do you remember this discussion, I think you and I may have discussed this.

**Comment by pgaudet** [comment]
Emailed Colin, Astrid, Marcio, Ruth and Val.

**Comment by pgaudet** [comment]
@krchristie There is some discussion in SF https://sourceforge.net/p/geneontology/ontology-requests/7557/
where the last comment proposes obsoletion. 

Pascale

**Comment by ukemi** [comment]
I believe this was reconsidered here, #7774,  and we redefined the term.

**Comment by krchristie** [comment]
Yes, the previous two comments make essentially the same argument as the current arguments, and it remains true that it is not valid to try to define the specific composition of an RNAP II holoenzyme complex within GO using either part_of or has_part relationships. 

However, the logic for keeping the term that was decided upon in the ticket that @ukemi referred to was that while it is not possible to define a singular composition of RNAP II holoeyzme within GO, there is still biological relevance for such a term as the entity that is active for initiation of transcription, and desire/utility for such a term within GO. Thus, it was decided to broaden the definition in such a way that it did not require a precise composition. Apparently, removing the invalid part_of links was forgotten.

Going forwards, it seems like there could be utility for such a general RNAP II holoenzyme term within GO-CAM models as it would allow annotators to indicate which specific initiation factors were present in the holoenzyme/PIC at the promoter of a specific gene being annotated.

Another tidbit,  my understanding is that holoenzyme has generally been used to refer to the RNAP plus initation factors when NOT associated with DNA, while the PIC is the RNAP plus initation factors WHILE associated with the DNA. I am not opposed to merging the holoenzyme and PIC terms as I don't really see differential need with respect to annotation for holoeyzme versus PIC.

I am less convinced that it is OK to obsolete the holoenzyme term because it is definitely not equivalent to the RNAP II core enzyme term, and the majority of things annotated to holoenzyme should definitely not end up annotated to the core term because then we will have lots of complaints of things which do not have RNA polymerase activity being annotated to that function.

**Comment by judyblake** [comment]
Just checking. Aren’t complexes in GO agreed to be defined by their function not the parts. Then the specific instance of the complex is defined in PRO with specific membership.

**Comment by krchristie** [comment]
@judyblake - yes, the previous decision to keep the holoenzyme term is consistent with definition by its biological role.

In this case, I don't think PRO will be able to define specific composition either as researchers don't create specific names for all the different variants of holoenzyme that they purify or otherwise encounter.

**Comment by pgaudet** [comment]
The suggestion by the transcription group is to merge holoenzyme into PIC (see #15870) 

- Pascale

**Comment by ValWood** [comment]
Action:

The suggestion by the transcription group is to merge holoenzyme into PIC (see https://github.com/geneontology/go-ontology/issues/15870)

is this still the action here?

**Comment by krchristie** [comment]
> Action:
> 
> The suggestion by the transcription group is to merge holoenzyme into PIC (see #15870)
> 
> is this still the action here?

I think it would be a bad idea to merge "RNAP II holoenzyme" into the PIC because the PIC by definition is bound to DNA while the RNAP II holoenzyme is not. The last I read, there was still debate about what the assembly pathway is, whether the holoenzyme assembles first and then binds DNA,  or whether it only assembles on DNA, or whether both pathways exist influenced by the exact sequence of the promoter (see Thomas MC, Chiang CM. The general transcription machinery and general cofactors. Crit Rev Biochem Mol Biol. 2006 May-Jun;41(3):105-78. PMID:16858867. https://pubmed.ncbi.nlm.nih.gov/16858867/ ).

I'm wondering if a reasonable solution would be to obsolete the "RNAP II holoenzyme" term and suggest the PIC term as a consider term.

-Karen

**Comment by krchristie** [comment]
FYI: The definition of the txn PIC seems possibly inadequate. It seems to imply that everything binds the promoter, while I think that some components do not bind directly. In addition, the referenced cited for the definition says that the presence of RNAP II is required for it to be called the PIC and the def does not indicate that.

transcription preinitiation complex - GO:0097550
Def: A protein-DNA complex composed of proteins binding promoter DNA to form the transcriptional preinitiation complex (PIC), the formation of which is a prerequisite for transcription.

**Comment by pgaudet** [comment]
Discussed with @colinlog 
Action points: 

- [ ] obsolete  GO:0016591 RNAP II holoenzyme  - with CONSIDER GO:0097550 transcription preinitiation complex 
- [ ] Move GO:0097550 transcription preinitiation complex  under GO:0055029  nuclear DNA-directed RNA polymerase complex
- [ ] Move all current children of GO:0016591 RNAP II holoenzyme under GO:0097550 transcription preinitiation complex 
- [ ] Change definition of GO:0055029  nuclear DNA-directed RNA polymerase complex to: "A large and dynamic assembly of general transcription factor complexes and RNA polymerase that associate with promoter DNA and is required for transcription initiation.

**Comment by pgaudet** [comment]
Annotations to have been removed/fixed, see https://docs.google.com/spreadsheets/d/1vIoh1iIGFg22FdZto1BYHv7UZIuMqzB2N_U-rdbSXFw/edit#gid=0

There is 1 InterPro mapping, IPR026213, see https://github.com/geneontology/go-annotation/issues/4405


Thanks, Pascale

**Comment by krchristie** [comment]
> Discussed with @colinlog Action points:
> 
>     * [ ]  obsolete  GO:0016591 RNAP II holoenzyme  - with CONSIDER GO:0097550 transcription preinitiation complex

Apologies that I did not catch this before, but it seems to me that it would be better to maintain the specificity of the relationship to RNAP II, which the current term "GO:0097550 transcription preinitiation complex" does not have. I think a better CONSIDER term would be to have a child term of "GO:0097550 transcription preinitiation complex" that is specific to RNAP II.

___

>     * [ ]  Move GO:0097550 transcription preinitiation complex  under GO:0055029  nuclear DNA-directed RNA polymerase complex
 
I'm not sure this works. See below for my comment on the fact that your new proposed definition for "GO:0055029  nuclear DNA-directed RNA polymerase complex"

___
>     * [ ]  Move all current children of GO:0016591 RNAP II holoenzyme under GO:0097550 transcription preinitiation complex

This term does not have any is_a children. Are you referring to the part_of relationships? Again, I think that the appropriate replacement term for relationships of the existing "GO:0016591 RNAP II holoenzyme" term would be to have a child term of "GO:0097550 transcription preinitiation complex" that is specific to RNAP II.

___
>     * [ ]  Change definition of GO:0055029  nuclear DNA-directed RNA polymerase complex to: "A large and dynamic assembly of general transcription factor complexes and RNA polymerase that associate with promoter DNA and is required for transcription initiation.

This proposed definition works for this term: "GO:0097550 transcription preinitiation complex', but NOT for the majority of current children  of "GO:0055029  nuclear DNA-directed RNA polymerase complex", which are NOT the large dynamic assemblies of GTFs and an RNAP, but are ONLY the RNAP core enzymes.

**Comment by ValWood** [comment]
It seems odd to make all of the PIC and "RNA polymerase complex", since the RNA polymerase is only part_of the PIC. Broadening the definition  seems odd, but this seems this change already been done:

https://www.pombase.org/term/GO:0030880
all of the regulatory factors are now "RNA polymerase". To me the RNA polymerases are the complexes I, II & III with RNA polymerase catalytic activity.

**Comment by ValWood** [comment]
OK it seems it has always been like this because 

GO:0055029   nuclear DNA-directed RNA polymerase complex
is a parent of 
GO:0016591   RNA polymerase II, holoenzyme
so maybe it's OK, ignore my comments.

**Comment by krchristie** [comment]
> OK it seems it has always been like this because
> 
> GO:0055029 nuclear DNA-directed RNA polymerase complex is a parent of GO:0016591 RNA polymerase II, holoenzyme so maybe it's OK, ignore my comments.

The current definition of **GO_0055029 - nuclear DNA-directed RNA polymerase complex**
**def:** A protein complex, located in the nucleus, that possesses DNA-directed RNA polymerase activity.

is broad enough to encompass ANY complex that has the capacity for RNA polymerase activity, so I never had an objection to including the holoenzyme term as well as the various RNAP core complexes. Note also that the holoenzyme was generally considered to be a free floating complex not associated with DNA.

To go with this proposal:
> Change definition of GO:0055029 nuclear DNA-directed RNA polymerase complex to: "A large and dynamic assembly of general transcription factor complexes and RNA polymerase that associate with promoter DNA and is required for transcription initiation.

- is narrowing the definition to the point that it is NO LONGER able to accomodate the non-DNA bound RNAP core complexes, because the core complexes are not necessarily bound to DNA.
- In addition, this proposed definition is using the phrase "general transcription factor", which is generally not used with either RNA polymerases I or III, so it seems inappropriately specific to RNA polymerase II.  
- Considering that this definition also includes the phrase "is required for initiation", I think that this definition is actually defining the RNAP II specific PIC, not an "RNA polymerase complex" generally.

Personally, I think that the existing term for "RNA polymerase complex" would be better kept specific to the RNA polymerase core enzymes, so that only annotations for RNA polymerase enzyme subunit will map up to this term in enrichment analyses. We may need to narrow the definition to make it clear if we want this term to only be for core RNAP complexes.

As I said previously, I think that what we really need as a consider term for **GO:0016591 RNAP II holoenzyme**  would be "**RNA polymerase II transcription preinitiation complex**", NOT  GO:0097550 transcription preinitiation complex since other RNA polymerases also have preintiation complexes (though we haven't created any terms for them in GO).


**This proposed new definition:** "A large and dynamic assembly of general transcription factor complexes and RNA polymerase that associate with promoter DNA and is required for transcription initiation.

is actually perfect for a term "**RNA polymerase II transcription preinitiation complex**", which I think we should create since it would be the appropriate target for a CONSIDER term for "RNA polymerase II, holoenzyme", that won't be inappropriately general if we ever have a need to create preinitiation complex terms for any other RNAP. 

I also think that the transcription preinitiation complex should keep its existing parentage. This is the component ontology after all, and as I said previously, I don't really think we want all of the various subunits of transcription initiation factors mapping up to the "RNA polymerase complex" term.
